doc/_templates/autosummary.rst

```diff
@@ -16,11 +17,13 @@
 
 {% if objtype in ['class', 'method', 'function'] %}
 {% if objname in ['AxesGrid', 'Scalable', 'HostAxes', 'FloatingAxes',
-                      'ParasiteAxesAuxTrans', 'ParasiteAxes'] %}
+'ParasiteAxesAuxTrans', 'ParasiteAxes'] %}
```

Comment:
That's because it thinks it's reStructuredText, but it's actually a jinja2 template with reStructuredText content. But I don't know how to tell the linter that.

Reply:
It seems rstcheck does not have file excludes 😞